tests/migration/online_migration/yast2_migration.pm

```diff
@@ -112,8 +112,8 @@ sub run {
         mouse_hide(1);
         assert_screen 'generic-desktop';
 
+        turn_off_screensaver();
```

Comment:
It is better to run more verification log such as online migration from SLES15*, so is for opensuse migration.

Reply:
http://openqa.nue.suse.com/tests/5893937  from SLES15
https://openqa.opensuse.org/t1711790        opensuse